ukpa/templates/ukpa/nomination.html

```diff
@@ -0,0 +1,16 @@
+{% extends 'ironcage/base.html' %}
+{% load avatar_tags %}
+
+{% block content %}
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+  {% avatar user %}<br>
+    <h4>{{ form.description.label }}</h4>
+    <div>{{ nomination.statement|linebreaks }}</div>
+    <hr />
+
+    <a href="{% url 'ukpa:nomination_delete' nomination.nomination_id %}" class="btn btn-danger">Withdraw your nomination</a>
```

Comment:
Suspect we'll have to do the same for things like withdrawing CFP proposals and grant applications.